wandb/integration/keras/keras.py

```diff
@@ -702,8 +702,8 @@ def on_train_begin(self, logs=None):
         if _can_compute_flops():
             try:
                 wandb.summary["GFLOPs"] = self.get_flops()
-            except Exception as e:
-                wandb.termwarn("Unable to compute FLOPs for this model.")
+            except:
+                pass
```

Comment:
I don't think it's a good practice to use `except: pass` for failing something silently. If it's not, can someone suggest a better approach?